meilisearch/models.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from typing import Any, List, Dict, Optional
+from camel_converter.pydantic_base import CamelBase
+
+@dataclass
+class MultiDocument:
```

Comment:
This class should be called `DocumentsResults`
```suggestion
class DocumentsResults:
```